framework/aster-frame/src/task/processor.rs

```diff
@@ -63,7 +70,7 @@ pub fn schedule() {
 
 pub fn preempt() {
     // disable interrupts to avoid nested preemption.
-    let disable_irq = disable_local();
```

Comment:
The simplest approach to fix the bug is to remove this line of code, isn't it? The original intention of this line is to make the `preempt` function robust even when our kernel threads can be preempted. But kernel preemption, as opposite to user preemption, is harder to implement. I think for the short term, we should focus on the easier goal of supporting user preemption.

Reply:
@fgh1999 Any opinion?